
Remove code to pick up LDFLAGS for swig; breaks for lib64.

--- subversion-0.24.2/build/ac-macros/swig.m4.swig
+++ subversion-0.24.2/build/ac-macros/swig.m4
@@ -69,20 +69,6 @@
       ])
       SWIG_LIBSWIG_DIR="$ac_cv_swig_swiglib_dir"
 
-      AC_CACHE_CHECK([if swig needs -L for its libraries],
-        [ac_cv_swig_ldflags],[
-        # The swig libraries are one directory above the
-        # `swig -swiglib` directory.
-        ac_cv_swig_ldflags=""
-        swig_lib_dir="`dirname $ac_cv_swig_swiglib_dir`"
-        if test "$swig_lib_dir" &&
-           test "$swig_lib_dir" != "/lib" &&
-           test "$swig_lib_dir" != "/usr/lib"; then
-          ac_cv_swig_ldflags="-L$swig_lib_dir"
-        fi
-      ])
-      SWIG_LDFLAGS="$ac_cv_swig_ldflags"
-
       AC_CACHE_CHECK([for Python includes], [ac_cv_python_includes],[
         ac_cv_python_includes="`$PYTHON ${abs_srcdir}/build/get-py-info.py --includes`"
       ])
